lib/internal/per_context/domexception.js

```diff
@@ -2,14 +2,23 @@
 
 const {
   Error,
+  FunctionPrototypeCall,
   ObjectDefineProperties,
   ObjectDefineProperty,
-  SafeWeakMap,
+  ObjectGetOwnPropertyDescriptors,
+  ObjectGetPrototypeOf,
+  ObjectSetPrototypeOf,
+  ReflectConstruct,
   SafeMap,
+  SafeWeakMap,
+  SymbolFor,
   SymbolToStringTag,
   TypeError,
 } = primordials;
 
+const kClone = SymbolFor('kClone');
+const kDeserialize = SymbolFor('kDeserialize');
```

Comment:
If these are going to start using `Symbol.for()`, then they should use the `nodejs.' prefix... so something like `SymbolFor('nodejs.clone')` and `SymbolFor('nodejs.deserialize')`

Reply:
Sure, done in https://github.com/nodejs/node/pull/41148/commits/1a8395eba12658303f0f1cf4e81f2bcd8686978f